### DISCUSSION TITLE
Why was my issue removed?

### DISCUSSION THREAD

**Discussion by dictav** [community/community#20497] [Projects and Issues]
Hi,

I found that some issues aren't reachable in my organization's repository.

For example, I created issue#444 in our repository before, but `https://github.com/myorg/myrepo/issues/444` returns a 404 error now.
However, in an issue comment, if I type `#444`, the issue will be auto-filled. Also, I can see the title. But the link is not generated, and I cannot open the issue.

Is something broken in our repository?

**Comment by 0x0is1** [comment]
Facing same issue.

**Comment by jfullstackdev** [comment]
> https://github.com/myorg/myrepo/issues/444

do you really own that Org named `myorg` ? I guess no, if you can send the link of the repo you are referring to, maybe I can take a look too
